src/Types/ReceiveName.cs

```diff
@@ -34,6 +34,18 @@ public static bool TryParse(string name, out (ReceiveName? ReceiveName, Validati
         return validate;
     }
 
+    /// <summary>
+    /// Get the contract name part of <see cref="Receive"/>.
+    /// </summary>
+    /// <returns>Contract name</returns>
+    public string GetContractName() => this.Receive[..this.Receive.IndexOf('.')];
+
+    /// <summary>
+    /// Get entrypoint part of <see cref="Receive"/> which is the entrypoint called on the contract.
+    /// </summary>
+    /// <returns>Entrypoint</returns>
+    public string GetEntrypoint() => this.Receive[(this.Receive.IndexOf('.') + 1)..];
```

Comment:
I think we should have an `Entrypoint` type, instead of using string.
We will need this when we add support for contract update transactions.